### PR TITLE
Test “fill” of square and circle mark

### PR DESCRIPTION
**Pull request by kanitw**

### CODE REVIEW

**Comment by kanitw** [comment]
Actually I forgot to add two more cases in the scaffolding:
- “It should have correct shape”  (test props.shape.value === ‘circle’ / ‘square’ for each mark type.

**Comment by domoritz** [comment]
When you're done, remove the [WIP] from the pr title.

**Comment by kanitw** [comment]
And tag us in the comment :)

File: test/compile/mark-point.test.ts [hunk 1]
```diff
@@ -114,7 +114,15 @@ describe('Mark: Square', function() {
   });
 
   it('should be filled by default', function() {
-    // TODO
+    const model = parseModel({
```
Comment (ayush29feb) [inline test/compile/mark-point.test.ts]: Shouldn't the cost model and props be defined outside it()? Looking at the rest of the tests I think it should be defined inside the describe()

Comment (domoritz) [inline test/compile/mark-point.test.ts]: Here, we want to test different models parsed from different specs. If we can extract some common functionality, we can of course do that.

**Comment by ayush29feb** [comment]
So both circle and square have this
```javascript
it('should return the correct mark type', function() {
    // TODO call square.markType()
});
```
but atom says `Supplied parameter do not match any signature of call target`

I have finished the rest of the test cases

**Comment by domoritz** [comment]
How do you call `sqare.markType`?

**Comment by ayush29feb** [comment]
I think it figured it out. Please check the commit I made in ayush/test-square branch

**Comment by domoritz** [comment]
e1f514f0e240f319aa57f56d5713f277af51bc55

**Comment by domoritz** [comment]
Instead of creating a model, you can just test `point.markType() === "symbol"`.

**Comment by ayush29feb** [comment]
`circle.markType()` & `sqaure.markType` doesn't seem to be working.

**Comment by domoritz** [comment]
Fixed in https://github.com/vega/vega-lite/pull/977. I'm glad you caught the bug!

**Pull request closed without merging**